spec/menu_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'menu'
+
+describe Menu do
+  let(:dish) { { name:'Burger', price: 8 } }
```

Comment:
I wonder if this could be a double rather than a variable?

Reply:
Good question. When we review the code let's give it a go and see what happens!